libraries/lib-stretching-sequence/tests/CMakeLists.txt

```diff
@@ -0,0 +1,31 @@
+add_compile_definitions(CMAKE_SOURCE_DIR="${CMAKE_SOURCE_DIR}")
+
+add_unit_test(
+   NAME
+      lib-stretching-sequence
+   SOURCES
+      AudioContainerHelper.h
+      AudioSegmentSampleViewTest.cpp
+      FloatVectorClip.cpp
+      FloatVectorClip.h
+      MockAudioSegmentFactory.h
+      MockSampleBlock.cpp
+      MockSampleBlock.h
+      MockSampleBlockFactory.cpp
+      MockSampleBlockFactory.h
+      MockPlayableSequence.h
+      SilenceSegmentTest.cpp
+      StretchingSequenceTest.cpp
+      StretchingSequenceIntegrationTest.cpp
+      WaveClipSegmentTest.cpp
+      TestWaveClipMaker.cpp
+      TestWaveClipMaker.h
+      TestWaveTrackMaker.cpp
+      TestWaveTrackMaker.h
+   MOCK_PREFS
+   MOCK_AUDIO
+   WAV_FILE_IO
```

Comment:
I think you do need MOCK_PREFS but why MOCK_AUDIO?

Reply:
`AudacityProject::Create()` leads to an audio-io setup in my `TestWaveTrackMaker`. Not sure if/how it could be avoided.